tutorials/Python/12_ProteinAnalysis/12_ProteinAnalysis.ipynb

```diff
@@ -0,0 +1,468 @@
+    "\n",
+    "We will only be giving a very superficial overview of MDAnalysis, if you want to know more, please have a look at the [MDAnalysis user guide](https://userguide.mdanalysis.org/1.0.0/index.html).\n",
+    "\n",
+    "One of the core components of MDAnalysis is the `Universe` class. You can consider this as the container where we store all the information about the structure file. In a PDB structure, this includes (amongst many other things); 3D coordinates for all the heavy atoms, atom names (i.e. pseudo-abitratry labels about the types of atoms in the structure), elements, residue names, chain identifiers, and temperature factors.\n",
+    "\n",
+    "First, let us create a `Universe` class and call it `pdb` by passing it a string with the path to our PDB file:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import MDAnalysis\n",
+    "\n",
+    "pdb = MDAnalysis.Universe('datafiles/1HSG.pdb')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `Universe` object has plenty of different attributes and methods, most of which we will not cover here.\n",
+    "\n",
+    "The main one that you will work with in the MD tutorial is `trajectory`, which allows you to traverse through a simulation trajectory. However since we only have a single PDB structure, we don't have to deal with this here.\n",
+    "\n",
+    "Let's use the `Universe` to gather some basic information about the 1HSG structure. Take some time to look at its [PDB entry](https://www.rcsb.org/structure/1hsg). From the page, we can see that the structure has a total of 1686 atoms, 198 residues, and two chains (called `A` and `B`). We can use MDAnalysis to recover this data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# We can get the number of atoms using the \"atoms\" sub-class\n",
+    "# \"atoms\" handles all the information about the atoms in a structure\n",
+    "# here it has a `n_atoms` attribute which tells you how many atoms there are\n",
+    "print(\"number of atoms: \", pdb.atoms.n_atoms)\n",
+    "\n",
+    "# We can also use `n_residues` to get the number of residues\n",
+    "print(\"number of residues: \", pdb.atoms.n_residues)\n",
+    "\n",
+    "# And `n_segments` for chains (MDAnalysis calls chains \"segments\")\n",
+    "print(\"number of chains\", pdb.atoms.n_segments)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As you probably noticed, the number of residues returned as 326, not 198. Why do you think this is?\n",
+    "\n",
+    "> Answer: the PDB page states the number of proteins, so there are 128 non-protein residues\n",
```

Comment:
Is the answer supposed to be here?

Same questions for the answers below.

Reply:
Yeah, since it's not code and we require it for the next bits I wanted to just have it there.